frontend/src/metabase/writeback/components/ActionCreator/ActionCreatorHeader.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { t } from "ttag";
+
+import type { ActionType } from "metabase/writeback/types";
+
+import {
+  Container,
+  LeftHeader,
+  EditableText,
+  SaveButton,
+  CompactSelect,
+} from "./ActionCreatorHeader.styled";
+
+type Props = {
+  name: string;
+  onChangeName: (name: string) => void;
+
+  type: ActionType;
+  onChangeType?: (type: ActionType) => void;
+
+  onCommit: () => void;
```

Comment:
WDYT about renaming it to `onSave`? Feel it'd be more consistent with other components across Metabase. I know it used be called like that in writeback's HTTP editor, but on a second thought I feel it might be a right thing to do